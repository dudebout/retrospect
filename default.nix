{ nixpkgs ? <nixpkgs> }:

with (import nixpkgs {});

callPackage (import ./package.nix) {
  inherit stdenv;
  inherit (emacsPackagesNg) melpaBuild dash org-plus-contrib;
}
