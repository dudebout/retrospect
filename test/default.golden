* Details
|-------------------+------+------+------|
| Bucket A          | 0:01 |      |      |
|-------------------+------+------+------|
| level 1           | 0:01 |      |      |
|     level 1A      |      | 0:01 |      |
|-------------------+------+------+------|
| Bucket B          | 0:02 |      |      |
|-------------------+------+------+------|
| level 2           | 0:02 |      |      |
|     level 2A      |      | 0:01 |      |
|         level 2Aa |      |      | 0:01 |
|     level 2B      |      | 0:01 |      |
|-------------------+------+------+------|
| Bucket C          | 0:02 |      |      |
|-------------------+------+------+------|
| level 1           | 0:01 |      |      |
|     level 1B      |      | 0:01 |      |
| level 2           | 0:01 |      |      |
|     level 2A      |      | 0:01 |      |
|-------------------+------+------+------|
